frontend/src/features/public-form/components/FormFooter/FormFooter.tsx

```diff
@@ -32,7 +36,7 @@ export const FormFooter = (): JSX.Element => {
         >
           <Box id={captchaContainerId} />
           <Box w="100%">
-            <AppFooter variant="compact" />
+            <AppFooter variant="compact" bgColorScheme={bgColor} />
```

Comment:
instead of bgColorScheme, we could use `containerProps` instead:

```tsx
<AppFooter 
  variant="compact" 
  containerProps={{
    bg,
  }} 
/>
```

Reply:
ohhh makes sense ok!